src/Symfony/Component/Cache/Adapter/AbstractAdapter.php

```diff
@@ -76,15 +76,15 @@ static function ($deferred, $namespace, &$expiredIds) use ($getId) {
                     $key = (string) $key;
                     if (null === $item->expiry) {
                         $ttl = 0 < $item->defaultLifetime ? $item->defaultLifetime : 0;
-                    } elseif (0 >= $ttl = (int) ($item->expiry - $now)) {
+                    } elseif (0 >= $ttl = (int) (0.1 + $item->expiry - $now)) {
```

Comment:
This could look like a "magic number" in the future and people won't understand why we use it. We could add a very brief comment explaining "why" ... or we could move this to a self-explanatory private constant (e.g. `self::MINIMUM_CACHE_TTL` or something better).

Reply:
The value is spread in many unrelated adapters, a private constant doesn't work.
I'm not sure this deserves more honestly, `git blame` can tell why if anyone wonders.